app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
+    {% for draft in drafts  %}
+    <li>
+      <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ draft.serviceName }}</a>
+    </li>
+    {% endfor %}
+    </ul>
+  {% endif %}
+  
   {% if framework.status == 'open' %}
-    {{ summary.heading("Complete services") }}
+      <h2 class="govuk-heading-m">Complete services</h2>
   {% elif framework.status == 'pending' or 'standstill' %}
     {% if declaration_status == 'complete' %}
-      {{ summary.heading("Submitted services") }}
+        <h2 class="govuk-heading-m">Submitted services</h2>
     {% else %}
-      {{ summary.heading("Completed services") }}
+        <h2 class="govuk-heading-m">Completed services</h2>
     {% endif %}
   {% endif %}
-  {% call(draft) summary.list_table(
-    complete_drafts,
-    caption="Complete services",
-    empty_message="You haven’t marked any services as complete yet." if framework.status == 'open' else "You didn’t mark any services as complete.",
-    field_headings=[
-        "Service name",
-        "Progress",
-        "Make a copy"
-    ],
-    field_headings_visible=False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(
-          draft.serviceName,
-          url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id)
-      ) }}
-      {% if framework.status == 'open' %}
-        {% call summary.field(action=True) %}
-          <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
-            {{ govukButton({
-              "text": "Make a copy",
-              "classes": "govuk-!-margin-0",
-            }) }}
-          </form>
-        {% endcall %}
-      {% else %}
-        {{ summary.text() }}
-      {% endif %}
-    {% endcall %}
-  {% endcall %}
 
-  <hr class="govuk-section-break govuk-section-break--m">
+  {% if not complete_drafts %}
+  {{ "You haven’t marked any services as complete yet." if framework.status == 'open' else "You didn’t mark any services as complete."  }}
+  {% endif %}
+  {% set ns = namespace(completed_rows = []) %}
+   {% if framework.status == 'open' %}
+    {% for complete_draft in complete_drafts %}
+    {% set service_link %}
+    <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ complete_draft.serviceName }}</a>
+    {% endset %}
+    {% set make_copy_button %}
+    <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=complete_draft.lot, service_id=complete_draft.id) }}">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
+        {{ govukButton({
+          "text": "Make a copy",
```

Comment:
```suggestion
          "html": 'Make a copy <span class="govuk-visually-hidden">of {{ complete_draft.serviceName }}</span>',
```

This will help screenreaders not just reel off "Make a copy" repeatedly.